hack/run-ose-tests.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+. $(dirname "$0")/common.sh
+
+export CONTAINER_MGMT_CLI="${CONTAINER_MGMT_CLI:-docker}"
+export TESTS_REPORTS_PATH="${TESTS_REPORTS_PATH:-/tmp/artifacts/}"
+export TESTS_TYPE="${TESTS_TYPE:-openshift/conformance/parallel}"
```

Comment:
-> TESTS_FILTER maybe?

Reply:
or, OSE_TESTS_FILTER

Reply:
they call it now origin-tests
because it's no longer ose..